TELCODOCS-1508: Doc improvements from SME feedback (#70756)
openshift/openshift-docs: networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc

toc::[]

> As a cluster administrator, you can configure an additional network for a virtual routing and forwarding (VRF) domain by using the CNI VRF plugin. The virtual network created by this plugin is associated with a physical interface that you specify. 

Reviewer: Spell out CNI on first appearance?

Reviewer: ```suggestion
As a cluster administrator, you can configure an additional network for a virtual routing and forwarding (VRF) domain by using the CNI VRF plugin. The virtual network that this plugin creates is associated with the physical interface that you specify. 
```

Author: Regarding CNI, I am tempted to leave this alone as it's part of the plugin name? Container Network Interface (CNI) VRF plugin is a bit confusing?

Reviewer: oh yeah if the name is the abbreviated version that makes sense to me so the user will find it :+1:

Using a secondary network with a VRF instance has the following advantages:

> Workload isolation:: Isolate workload traffic by configuring the additional network's VRF instance. 

Reviewer: ```suggestion
Workload isolation:: Isolate workload traffic by configuring a VRF instance for the additional network.
```

Improved security:: Enable improved security through isolated network paths in the VRF domain.

> Multi-tenancy support:: Support multi-tenancy through network segmentation with a unique routing table in the VRF domain for each tenant.

Reviewer: ISG says not to use a hyphen here, but I imagine that's a common word in this content area. Would leave it alone for consistency, but consider updating the overall content to use "multitenancy" as a separate effort.

[NOTE]
====

> Applications that use VRFs need to bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. `SO_BINDTODEVICE` binds the socket to a device that is specified in the passed interface name, for example, `eth1`. To use `SO_BINDTODEVICE`, the application must have `CAP_NET_RAW` capabilities.

Reviewer: ```suggestion
Applications that use VRFs must bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. The `SO_BINDTODEVICE` option binds the socket to the device that is specified in the passed interface name, for example, `eth1`. To use the `SO_BINDTODEVICE` option, the application must have `CAP_NET_RAW` capabilities.
```
